pkg/commands/clusterstack/save.go

```diff
@@ -63,7 +63,7 @@ kp clusterstack save my-stack --build-image ../path/to/build.tar --run-image ../
 				return err
 			}
 
-			return update(ctx, authn.DefaultKeychain, cStack, buildImageRef, runImageRef, factory, ch, cs, w)
+			return patch(ctx, authn.DefaultKeychain, cStack, buildImageRef, runImageRef, factory, ch, cs, w)
```

Comment:
Github doesn't allow commenting on unchanged lines ([yet](https://github.com/github/feedback/discussions/4452)). The lines I'm commenting on are [28 and 29](https://github.com/vmware-tanzu/kpack-cli/pull/243/files#diff-88caecb88f9ce416ca4bb01723a00a81e1895e88f6065785e30c755e100967b9R28-R29) of this file. Instead of "Create or update a cluster stack", should we use "Create or patch a cluster stack"?